src/js/components/AppVersionComponent.jsx

```diff
@@ -134,6 +135,16 @@ var AppVersionComponent = React.createClass({
       ? <UnspecifiedNodeComponent />
       : <dd><pre>{JSON.stringify(appVersion.container, null, 2)}</pre></dd>;
 
+    var dependenciesNode = (appVersion.dependencies.length === 0)
```

Comment:
I suppose we don't check for `appVersions.dependencies == null` because they were introduced long time ago and so we're sure it's always an array?